include/oneapi/tbb/concurrent_queue.h

```diff
@@ -131,8 +133,43 @@ class concurrent_queue {
         queue_allocator_traits::destroy(my_allocator, my_queue_representation);
         r1::cache_aligned_deallocate(my_queue_representation);
     }
+  
+    concurrent_queue& operator=( const concurrent_queue& other ){
```

Comment:
Ideally, special allocator support should be done for copy/move assignment, copy constructor and swap, but it can be a complicated in case of `concurrent_queue` that uses PIMPL implementation. But I guess we should at least add some TODO comments. Something like:
`//TODO: implement support for std::allocator_traits::propogate_on_container_copy_assignment` and so on.